frontend/app/templates/project/show/transportation/existing-conditions.hbs

```diff
@@ -1,7 +1,9 @@
 {{! transportation/existing-conditions controller template }}
 <div class="row">
   <div class="sixteen wide column">
-    {{transportation/study-area-map}}
+    <Transportation::StudyAreaMap
+      @analysis={{analysis}}
```

Comment:
I would opt for using the full model name here `transportationAnalysis` - it was redundant because we are scoped to the transportation route already, but downstream components refer to specific properties on that model and I don't want anyone to think that "analysis" is some _other_ kind of object. Thoughts?

Reply:
actually on second thought, i dont thumbs up. down stream, the component that updated the model is also scoped to transportation. I think it is clear that analysis in the context of transportation is transportation analysis